src/xtgeo/surface/_regsurf_import.py

```diff
@@ -250,13 +251,46 @@ def _import_irap_ascii(mfile):
     return args
 
 
-def import_ijxyz(mfile, template=None, **_):
+def import_ijxyz(mfile, template=None, engine="python", **kwargs):
     """Import OW/DSG IJXYZ ascii format."""
-    return _import_ijxyz_tmpl(mfile, template) if template else _import_ijxyz(mfile)
 
+    # override engine for this format as cxtgeo read will fail for memstreams, and the
+    # python implementation here is faster and more robust
+    use_engine = "python"
+    logger.debug("Override engine for ijxyx format; set to python")
 
-def _import_ijxyz(mfile):
-    """Import OW/DSG IJXYZ ascii format."""
+    if use_engine == "cxtgeo":  # will not be invoked in practice; remove in xtgeo 4
```

Comment:
Why not just remove the cxtgeo parts now? I think it's okay to makes these behind-the-scenes changes where the output will be the same regardless